templates/ces_client/about.html

```diff
@@ -0,0 +1,76 @@
+        <p>View shelter options and next steps <a href="http://www.suburbancook.org/emergency" target="_blank">by clicking here</a>.</p>
+
+        <h5 class='mt-5 font-weight-bold'>Shelter Access In Chicago</h5>
+        <p>Please visit any hospital emergency room or police station in Chicago, or call 311 and request transportation to shelter.</p>
+        <div class='row'>
+            <div class='col'>
+                <span class='font-weight-600'><i class="fas fa-user"></i> Single individuals can request shelter in person at the Chicago Garfield Service Center.</span><br> 
+                10 S Kedzie Ave<br>
+                Chicago, IL 60612<br>
+                Monday through Friday, 9am – 5pm
+            </div>
+            <div class='col'>
+                <span class='font-weight-600'><i class="fas fa-users"></i> Families with children can request shelter at the Salvation Army Shield of Hope at any time.</span><br> 
+                924 N Christiana Ave<br>
+                Chicago, IL 60651
+            </div>
+        </div>
+
+        <h5 class='mt-5 font-weight-bold'>Useful Links</h5>
+        <p><a href="https://www.csh.org/chicagoces" target="_blank">Chicago Coordinated Entry System</a></p>
+        <p><a href="https://allchicago.org" target="_blank">All Chicago: Making Homelessness History</a></p>
+        <p><a href="http://myentrypoint.org/" target="_blank">Suburban Coordinated Entry System</a></p>
+        <p><a href="http://www.suburbancook.org/" target="_blank">Alliance to End Homelessness in Suburban Cook County</a></p>
+
+        <h5 class="mt-5 mb-3 font-weight-bold">About the Bravely Engineered Navigator (B.E.N.)</h5>
+        <div class='row'>
+            <div class='col'>
+                <p>B.E.N. represents a robot who knows a lot about the homeless experience. He helps personalize the survey-taking process. <a href="https://datamade.us/" target="_blank">DataMade (a civic technology company in Chicago)</a> designed B.E.N. and "Connect to End Homelessness" in partnership with the <a href="https://www.csh.org" target="_blank">Corporation for Supportive Housing (a non-profit that advances supportive housing solutions)</a>.</p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='row'>
+    <div class='col-8 offset-2'>
+        <div class='row'>
+            <div class='col pt-3'><a class="img-fluid float-left mr-0" href="https://www.csh.org" target="_blank"><img class="img-width-sm" src="{% static 'images/CSH_Logo.svg' %}"></a></div>
+            <div class='col'><a class="img-fluid float-left mr-0" href="{% url 'index' %}"><img class="img-width-sm" src="{% static 'images/ben-icon-03.svg' %}" width='100'></a></div>
+            <div class='col pt-3'><a class="img-fluid float-left mr-0" href="https://datamade.us/" target="_blank"><img class="img-width-md" src="{% static 'images/datamade_logo.png' %}" width='180'></a></div>
+        </div>
+    </div>
+</div>
```

Comment:
This is what it looks like on mobile now:
<img width="479" alt="Screen Shot 2019-07-03 at 10 23 05 AM" src="https://user-images.githubusercontent.com/6961258/60604145-a61dff00-9d7c-11e9-99d4-690f1b7e5197.png">

I'd suggest stacking them into one column (i.e. one on top of each other) or reducing the space between each logo. If this is finicky, reverting to hiding them is okay.

Reply:
Got them to stack!